DTML.EduBot/Dialogs/StartLessonPlanDialog.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Luis.Models;
+
+namespace DTML.EduBot.Dialogs
+{
+    public partial class ChitChatDialog : QnaLuisDialog<object>
+    {
+        private LessonPlanDialog _lessonPlanDialog;
+
+        public ChitChatDialog(LessonPlanDialog lessonPlanDialog)
+        {
+            _lessonPlanDialog = lessonPlanDialog;
+        }
+
+        [LuisIntent("LessonPlan")]
```

Comment:
The intent name is "LearnEnglish", not "LessonPlan".

Let me give you some examples that will trigger LearnEnglish:

1. studytime
2. study time
3. exercises please
4. let's learn english
5. i want to improve my english

Note it is case-sensitive, i.e. has to be lower-case.